src/scripts/plugin/components/ads-label-loader.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+if (videojs.Component) {
```

Comment:
It's just the difference between VideoJS 4 and VideoJS 5 syntax for doing the same thing, apparently.